examples/pytorch/rgcn/entity_sample_multi_gpu.py

```diff
@@ -101,6 +102,7 @@ def main(args, devices):
     g.create_formats_()
 
     n_gpus = len(devices)
+    mp.set_start_method('spawn')
```

Comment:
I think we should add a comment like:
```
# required for mp.Queue() to work with mp.spawn()
```
as otherwise I think it will be easy for folks to assume this line is redundant since `mp.spawn()` uses the `spawn` start method.